guide/anatomy.rst

```diff
@@ -111,25 +111,38 @@ Other motors and sensors have similar conventions.
 Robot drivetrain control
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-For standard types of drivetrains (2 or 4 wheel, and mecanum), you'll want to
-use the :class:`.RobotDrive` class to control the motors instead of writing
-your own code to do it. When you create a RobotDrive object, you either specify
-which PWM channels to automatically create a motor for::
+For standard types of drivetrains (2 or 4 wheel, mecanum, kiwi), you'll want to
+use the various included class to control the motors instead of writing
+your own code to do it. For most standard drivetrains, you'll want to use one
+of three classes.
 
-	self.robot_drive = wpilib.RobotDrive(0,1)
+* :class:`wpilib.drive.DifferentialDrive` for differential drive/skid-steer drive platforms such as the Kit of Parts drive base, "tank drive", or West Coast Drive.
```

Comment:
Seems that since this is aimed for beginners, should explicitly call out 2/4 wheel, particularly since it's mentioned above.